projects/quickjs/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# build quickjs
+cd quickjs
+# Makefile should not override CFLAGS
+sed -i -e 's/CFLAGS=/CFLAGS+=/' Makefile
+if [ "$ARCHITECTURE" = 'i386' ]; then
+    export CFLAGS="$CFLAGS -m32"
```

Comment:
I think this will be automatically added by https://github.com/google/oss-fuzz/blob/f76ab1ea94f159c74e21b92350cf10c7c0605fcc/infra/base-images/base-builder/compile#L31

Reply:
Indeed, I got confused with `CFLAGS=` from Makefile